internal/driver/constants.go

```diff
@@ -29,6 +29,9 @@ const (
 	FrameRateValueDenominator       = "FrameRateValueDenominator"
 	FrameRateValueNumerator         = "FrameRateValueNumerator"
 	PathIndex                       = "PathIndex"
+	PixFmtWidth                     = "Width"
+	PixFmtHeight                    = "Height"
+	PixFmtPixFmt                    = "PixelFormat"
```

Comment:
this should probably just be: `PixelFormat = "PixelFormat"` regardless of the two above it

Reply:
This leads to duplicate variable but I will change to something better.